### DISCUSSION TITLE
dbt-core logs in third party servers if any.

### DISCUSSION THREAD

**Discussion by durbinar** [dbt-labs/dbt-core#7331] [General]
have noticed that dbt-core (v 1.4.5) tries to connect to server host: `fishtownanalytics.sinter-collect.com` upon failure, can some one clarify what dbt-core tries to log in this server and confirm if connection info is secured and protected ?, Thanks

**Comment by jtcohen6** [comment]
Hi @durbinar, this is for our anonymous usage tracking. It helps us understand how the product is being used, and informs future development: feature-specific investments, performance speedups, etc. The connection is via HTTPS.

You can see the full set of events included in our telemetry here: https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/tracking.py

And you can opt out: https://docs.getdbt.com/reference/global-configs#send-anonymous-usage-stats
